datafusion-cli/Cargo.toml

```diff
@@ -47,6 +47,7 @@ datafusion = { workspace = true, features = [
     "encoding_expressions",
     "nested_expressions",
     "parquet",
+    "parquet_encryption",
```

Comment:
can we please also add this to the list of crate features here:
- https://github.com/apache/datafusion?tab=readme-ov-file#crate-features